--- Python-3.4.3/Modules/Setup.dist	2016-06-11 19:53:12.545185032 +0800
+++ Python-3.4.3-android/Modules/Setup.dist	2016-06-11 19:53:27.453184765 +0800
@@ -358,7 +358,7 @@ _symtable symtablemodule.c
 # Andrew Kuchling's zlib module.
 # This require zlib 1.1.3 (or later).
 # See http://www.gzip.org/zlib/
-#zlib zlibmodule.c -I$(prefix)/include -L$(exec_prefix)/lib -lz
+zlib zlibmodule.c -I$(prefix)/include -L$(exec_prefix)/lib -lz
 
 # Interface to the Expat XML parser
 #
